va/http.go

```diff
@@ -212,17 +215,17 @@ func (va *ValidationAuthorityImpl) newHTTPValidationTarget(
 	path string,
 	query string) (*httpValidationTarget, error) {
 	// Resolve IP addresses for the hostname
-	addrs, err := va.getAddrs(ctx, host)
-	if err != nil {
-		return nil, err
-	}
-
+	addrs, resolvers, err := va.getAddrs(ctx, host)
```

Comment:
Still need to return `nil, err` here if we got an error. Returning both an error and a meaningful object at the same time is very non-idiomatic.